### PR TITLE
feat(components): Divider component

### PR DESCRIPTION
**Pull request by eddysims**
## Motivations

We need a way to visually divide sections of content.

## Changes

Adds a Divider component.

### Added

- Divider component

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/Divider/Divider.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import classnames from "classnames";
+import styles from "./Divider.css";
+
+interface DividerProps {
+  /**
+   * Width of the divider
+   */
```
Comment (seanhealy) [inline packages/components/src/Divider/Divider.tsx:8]: Would be good to specify the default here.

File: packages/components/src/Divider/Divider.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.divider {
+  height: var(--space-minuscule);
+  margin-bottom: var(--space-large);
```
Comment (seanhealy) [inline packages/components/src/Divider/Divider.css:3]: I wonder if divider should define any margin or just be tiny and expected to be within a `<Content>` or something… 🤔

**Comment by JeffMarvin** [comment]
@eddysims Only a small suggestion for you to consider!

**Comment by eddysims** [comment]
@JeffMarvin I have updated the prop description

**Pull request merged**